Index: todolist/src/test/java/com/nexai/todolist/TodolistApplicationTests.java
===================================================================
diff --git a/todolist/src/test/java/com/nexai/todolist/TodolistApplicationTests.java b/todolist/src/test/java/com/nexai/todolist/TodolistApplicationTests.java
deleted file mode 100644
--- a/todolist/src/test/java/com/nexai/todolist/TodolistApplicationTests.java	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
+++ /dev/null	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
@@ -1,13 +0,0 @@
-package com.nexai.todolist;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class TodolistApplicationTests {
-
-	@Test
-	void contextLoads() {
-	}
-
-}
Index: todolist/src/test/java/com/nexai/todolist/application-test.properties
===================================================================
diff --git a/todolist/src/test/java/com/nexai/todolist/application-test.properties b/todolist/src/test/java/com/nexai/todolist/application-test.properties
deleted file mode 100644
--- a/todolist/src/test/java/com/nexai/todolist/application-test.properties	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
+++ /dev/null	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
@@ -1,7 +0,0 @@
-spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=sa
-spring.datasource.password=password
-spring.datasource.platform=h2
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.jpa.hibernate.ddl-auto=create-drop
\ No newline at end of file
Index: todolist/src/test/java/com/nexai/todolist/repository/TaskRepositoryTest.java
===================================================================
diff --git a/todolist/src/test/java/com/nexai/todolist/repository/TaskRepositoryTest.java b/todolist/src/test/java/com/nexai/todolist/repository/TaskRepositoryTest.java
deleted file mode 100644
--- a/todolist/src/test/java/com/nexai/todolist/repository/TaskRepositoryTest.java	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
+++ /dev/null	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
@@ -1,35 +0,0 @@
-package com.nexai.todolist.repository;
-
-
-import com.nexai.todolist.entity.Task;
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-@DataJpaTest
-public class TaskRepositoryTest {
-    @Autowired
-    private TaskRepository taskRepository; // Репозиторий для работы с задачами
-
-    @Test
-    public void testSaveAndFindTask() {
-
-        Task task = new Task();
-        task.setTitle("Test Task");
-        task.setDescription("This is a test task.");
-        task.setStatus("Not Completed");
-
-        Task savedTask = taskRepository.save(task);
-
-        assertThat(savedTask).isNotNull();
-        assertThat(savedTask.getId()).isNotNull();
-
-        Task foundTask = taskRepository.findById(savedTask.getId()).orElse(null);
-
-        assertThat(foundTask).isNotNull();
-        assertThat(foundTask.getTitle()).isEqualTo("Test Task");
-        assertThat(foundTask.getDescription()).isEqualTo("This is a test task.");
-    }
-}
Index: todolist/src/test/java/com/nexai/todolist/service/TaskServiceTest.java
===================================================================
diff --git a/todolist/src/test/java/com/nexai/todolist/service/TaskServiceTest.java b/todolist/src/test/java/com/nexai/todolist/service/TaskServiceTest.java
deleted file mode 100644
--- a/todolist/src/test/java/com/nexai/todolist/service/TaskServiceTest.java	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
+++ /dev/null	(revision 119e9df606224ac6d46198586c16e4a905ac27e0)
@@ -1,30 +0,0 @@
-package com.nexai.todolist.service;
-
-import com.nexai.todolist.entity.Task;
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-
-import java.time.LocalDate;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
-@SpringBootTest
-public class TaskServiceTest {
-
-    @Autowired
-    private TaskService taskService;
-
-    @Test
-    void testCreateTask() {
-        Task task = new Task();
-        task.setTitle("Test Task");
-        task.setDueDate(LocalDate.of(2024, 12, 10));
-
-        Task createdTask = taskService.createTask(task);
-
-        assertNotNull(createdTask);
-        assertEquals("Test Task", createdTask.getTitle());
-    }
-}
